src/__tests__/utils.test.ts

```diff
@@ -1,29 +1,290 @@
-import type { Repository } from '../models'
+import type { Content } from 'mdast'
 
-import { mapRepositoryToQueryParams } from '~/utils'
+import type { Release, Repository, RepositoryQueryParams } from '~/models'
+import {
+  compareReleaseGroupsByPriority,
+  compareReleasesByVersion,
+  filterReleasesByVersionRange,
+  getMdastContentNodeTitle,
+  getMdastContentReleaseGroup,
+  getReleaseVersion,
+  isStableRelease,
+  mapRepositoryToQueryParams,
+  mapStringToRepositoryQueryParams,
+} from '~/utils'
 
 describe('mapRepositoryToQueryParams util', () => {
-  it('should map an undefined repository', () => {
-    const result = mapRepositoryToQueryParams()
+  it.each`
+    label               | input                                       | output
+    ${'an empty repo'}  | ${undefined}                                | ${{ owner: '', repo: '' }}
+    ${'a full repo'}    | ${{ owner: { login: 'foo' }, name: 'bar' }} | ${{ owner: 'foo', repo: 'bar' }}
+    ${'a partial repo'} | ${{ owner: {}, name: 'bar' }}               | ${{ owner: '', repo: 'bar' }}
+  `(
+    'should map $label',
+    ({
+      input,
+      output,
+    }: {
+      input: undefined | Repository
+      output: RepositoryQueryParams
+    }) => {
+      const result = mapRepositoryToQueryParams(input)
```

Comment:
I've moved these tests to `it.each` as spoken in the previous PR.

Reply:
The table layout makes the test easier to understand, I can see what the input and outputs are.

I did not know you could use a `label` with that `it.each` test construct. Very nice! 😄